next-env.d.ts

```diff
@@ -1,6 +1,5 @@
 /// <reference types="next" />
 /// <reference types="next/image-types/global" />
-/// <reference types="next/navigation-types/compat/navigation" />
```

Comment:
Why did this get changed?

Reply:
Weird. I don't see that locally. Sorry about that